lnrpc/autopilotrpc/autopilot.proto

```diff
@@ -26,72 +26,72 @@ option go_package = "github.com/lightningnetwork/lnd/lnrpc/autopilotrpc";
 // state of the daemon's autopilot agent, and also supply it with information
 // that can be used when deciding where to open channels.
 service Autopilot {
-    /* lncli: `autopilot status`
+  /* lncli: `autopilot status`
```

Comment:
We don't want to re-format the protos! This must be configurable somehow. See `make rpc-format` that uses `lnrpc/.clang-format` to format the protos. Those two things should not conflict.

Reply:
Thanks this makes sense!

Reply:
The linting failed, and, truth be told, I don't grasp the error message!
```
find . -name "*.proto" -print0 | \
xargs -0 "/tmp/protolint" lint
/tmp/protolint: 1: Syntax error: word unexpected (expecting ")")
make: *** [Makefile:265: protolint] Error [12](https://github.com/lightningnetwork/lnd/actions/runs/7516002369/job/20460598778?pr=8314#step:7:13)3
Error: Process completed with exit code 2.
```

Reply:
I'm also wondering if there's any way we could implement selective linting for changed files when running `make lint`. Currently, it takes a significant amount of time on my machine. Therefore, every time I want to check if I'm linting correctly, I have to push it to the branch to run it in the CI.

Reply:
Not sure where the error comes from. Why is the xargs approach necessary? Won't protolint only look at proto files in the first place?
IMO we don't need to support only looking at changed files unless the whole CI step otherwise takes more than 5 minutes (which I assume it doesn't).